### ISSUE TITLE
CustomImportOrder: no expected violations for empty line within import group

### ISSUE THREAD

**Issue by gmshake** [checkstyle/checkstyle#7119]
```
/var/tmp $ javac CustomImportOrder2.java
```

```
/var/tmp $ cat config.xml
```
```xml
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN" 
"http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">

    <module name="CustomImportOrder">
      <property name="customImportOrderRules"
               value="STANDARD_JAVA_PACKAGE"/>
    </module>

  </module>
</module>
```

```
/var/tmp $ cat CustomImportOrder2.java
```
```java
package com.example;

import java.util.Arrays;
import java.util.HashMap;

import java.util.LinkedList; // should warn
import java.util.List;

class CustomImportOrder2 {
}
```

```
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.24-all.jar \
       -c config.xml CustomImportOrder2.java
Starting audit...
Audit done.
```

---------------

The line commented with 'should warn' should trigger violations.
because in Google style there is clear definition that no blank lines should be between imports in same group - https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s3.3.3-import-ordering-and-spacing

> If there are both static and non-static imports, a single blank line separates the two blocks. There are no other blank lines between import statements.

**Comment by gmshake** [comment]
Related issue #3551. The implementation of #3551 was not well done thus causing #6680 and this one.

**Comment by gmshake** [comment]
@romani Update the input to prevent messing up with #7314 .

**Comment by rnveach** [comment]
Fix was merged
